new/LIT6374Confessions.xml

```diff
@@ -0,0 +1,82 @@
+                <title xml:lang="am" corresp="#t1" type="normalized">lakrǝstiyān hulu tasfā nuzzāze yammihon ṭǝqit q̄al</title>
+                <title xml:lang="en" corresp="#t1">A short word for all Christians as a confession of hope</title>
+                <title xml:lang="en" corresp="t2">Collection of Biblical Confessions from two monks from Dabra Bezan Monastery </title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="RL"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Theology"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="RL" when="2021-02-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"/>
+            <div type="edition">
+              <div type="textpart" subtype="incipit">
+                <note>The following are taken from EMIP2259 ff.33r</note>
+                <ab>
+                  <foreign xml:lang="gez">
+                    በስመ፡ አብ፡ <gap reason="ellipsis"/>
+                    ነዌጥን፡ በረድኤተ፡ እግዚአብሔር፡ ጽሒፈ፡ መጽሐፈ፡ አስተጉቡዕ፡ ዘአምብሉይ፡ ወሐዲስ፡ ዘአስተጋብእዎ፡ አኃው፡ ፍቁራን፡ መንኮሳት፡ ዘደብረ፡ ቤዛን፡
+                    አባ፡ ገብር፡ እግዚብሔር፡ ወአባ፡ ወልደ፡ አብርሃም፡ ርቱዓነ፡ ሃይማኖት፡ ይጽሐፍ፡ ሰሞሙ፡ እግዚአብሔር፡ ምስለ፡ ኵሎሙ፡ ደቂቀ፡ ቤዛን፡ አሜን።</foreign>
+                  <foreign xml:lang="am">
+                    ሰመ፡ ሥላሴን፡ መሠረት፡ አድርገን፡ ከብሉይ፡ ከሐዲስ፡ አወጣጥተን፡ ለክርስቲያን፡ ሁሉ፡ ተስፋ፡ ኑዛዜ፡ የሚሆን፡ ጥቂት፡ ቃል፡ እንጽፋለን፡ ተስፋውም፡ በጌታችን፡ በኢየሱስ፡ ክርስቶስ፡
```

Comment:
please check it the word is አውጥተን

Reply:
Again in the image above it is አወጣጥተን. I do see in Kane's dictionary that አወጣጠነ is an alternative to አዋጠነ both defined as 'to lay out a plan'